nepal_compliance/nepal_compliance/report/purchase_return_vat_register/purchase_return_vat_register.js

```diff
@@ -1,5 +1,5 @@
-// Copyright (c) 2024, njsubedi, Mukesh and contributors
-// For license information, please see license.txt
+// Copyright (c) 2025, Yarsa Labs Pvt. Ltd. and contributors
```

Comment:
Please use 2024, as the year should match when the code was created, not when the correction was made.